src/util/pointer_predicates.cpp

```diff
@@ -205,3 +205,17 @@ exprt object_lower_bound(
 
   return binary_relation_exprt(std::move(p_offset), ID_lt, std::move(zero));
 }
+
+exprt validate_all_derefs(const exprt &expr, const namespacet &ns)
```

Comment:
`s/validate_all_derefs/validate_all_dereferences/g`

Reply:
We should move this to `contracts/utils.cpp`.

Reply:
Done.